corehq/motech/repeaters/management/commands/populate_repeatrecords.py

```diff
@@ -50,7 +50,7 @@ def sql_may_have_next_check():
         from ...models import State
         is_very_old_pending_record = (
             'registered_at' not in couch
-            and get_state(couch) == State.Pending
+            and (get_state(couch) == State.Pending or get_state(couch) == State.Fail)
```

Comment:
I went with it because `get_state()` should involve only hash lookups, where `VALUE in COLLECTION` involves allocating and populating a collection, then iterating over it, which I think is more expensive (although admittedly, I have not measured). To optimize it a bit more I could cache the value of `get_state()` in a local variable, but it didn't seem worth it.